andela/aragon-cfh
# #141404583 users should receive a jwt on successful signup

#### What does this PR do?
It displays an error message whenever there is an error in signup or login

#### How should this be manually tested?
- You can try signing up with an existing user and see an error message displayed
- You can try signing up with incomplete fields and see an additional error message displayed
- You can also try signing in with an invalid username or password and see an error message displayed
- You can try signing up or signing in appropriately and be directed to the right page/url.
